File: FlatBuffersWindows/FlatBuffersWindows/model/vo/json/ApartamentoJSON.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FlatBuffersWindows.model.vo.json
{
    class ApartamentoJSON : ImovelJSON
    {
        public int numApto { get; set; }
        public float area { get; set; }
    }
}

File: FlatBuffersAndroid/app/src/main/java/com/talkitbr/flatbuffersandroid/model/vo/json/ApartamentoJSON.java
package com.talkitbr.flatbuffersandroid.model.vo.json;

public class ApartamentoJSON extends ImovelJSON {
    private float area;
    private int numApto;

    public float getArea() {
        return area;
    }

    public void setArea(float area) {
        this.area = area;
    }

    public int getNumApto() {
        return numApto;
    }

    public void setNumApto(int numApto) {
        this.numApto = numApto;
    }
}
